ext/mbstring/mbstring.c

```diff
@@ -2941,6 +2941,144 @@ PHP_FUNCTION(mb_strtolower)
 	RETURN_STR(mbstring_convert_case(PHP_UNICODE_CASE_LOWER, ZSTR_VAL(str), ZSTR_LEN(str), enc));
 }
 
+typedef enum {
+	MB_LTRIM = 1,
+	MB_RTRIM = 2,
+	MB_BOTH_TRIM = 3
+} MB_TRIM_MODE;
```

Comment:
Style nit: I think we usually write type names in lowercase (but keep MB_LTRIM etc in uppercase because they're constants).

Reply:
Thanks, fixed lowercase.